RecoPixelVertexing/PixelTrackFitting/test/BuildFile.xml

```diff
@@ -11,6 +11,15 @@
 <use   name="RecoTracker/TkTrackingRegions"/>
 <use   name="RecoPixelVertexing/PixelTriplets"/>
 <use   name="RecoPixelVertexing/PixelTrackFitting"/>
+<use name="cuda"/>
+<use name="cuda-api-wrappers"/>
```

Comment:
Could these be moved inside the `<bin></bin>` block? IIUC then they are applied only to the `testEigenGPU_t` and not everything (ok, doesn't matter too much in practice yet, but less to clean up later).